xbmc/interfaces/legacy/ModuleXbmcplugin.h

```diff
@@ -438,6 +439,33 @@ namespace XBMCAddon
     void setProperty(int handle, const char* key, const String& value);
 #endif
 
+#ifdef DOXYGEN_SHOULD_USE_THIS
+    ///
+    /// \ingroup python_xbmcplugin
+    /// @brief \python_func{ xbmcplugin.getWatchedMode(handle, content) }
+    ///-------------------------------------------------------------------------
+    /// Get the watched mode for given content type:
```

Comment:
the description is not well explained,  get watched mode for what?
this get the current filter mode of current directory

and the type of return value must be placed in `@return` key description